installer/pkg/workflow/destroy.go

```diff
@@ -1,5 +1,28 @@
 package workflow
 
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
+)
+
+var (
+	kubeconfigPath = filepath.Join("generated", "auth", "kubeconfig")
```

Comment:
This path should be built from the existing constants for these path segments.

Reply:
Fixed.